modules/installation-and-upgrade/pages/proxy-container-k3s-installation.adoc

```diff
@@ -105,6 +105,13 @@ services:
     metallb.universe.tf/loadBalancerIPs: 192.168.122.241
 ----
 
+
+[NOTE]
+====
+You need to adjust the parameter [literal]``metallb.universe.tf/loadBalancerIPs``to your network setup.
```

Comment:
```suggestion
You need to adjust the parameter [literal]``metallb.universe.tf/loadBalancerIPs``to your network setup.
You don't need to change the value in [literal]``metallb.universe.tf/allow-shared-ip`, in fact it will work with any kind of text value.
```